src/mixins/common-mixin.ts

```diff
@@ -86,23 +86,38 @@ export default class CommonMixin extends Vue {
 
   /** The appropriate edit label for corrections, alterations, change or conversion filings. */
   get editLabel (): string {
-    if (this.isCorrectionFiling) return 'Correct'
-    if (this.isAlterationFiling || this.isChangeFiling || this.isConversionFiling) return 'Change'
-    return 'Edit'
+    switch (true) {
+      case this.isCorrectionFiling: return 'Correct'
+
+      case this.isAlterationFiling:
+      case this.isChangeRegFiling:
+      case this.isConversionFiling: return 'Change'
+    }
```

Comment:
I think I prefer this layout for checking filings types: we know that we have 4 types, and here we can easily see that we're handling all 4 types.

Maybe I should change line 96 to log/throw an exception, since it means we forgot to handle a new filing type... Thoughts?